src/main/java/edu/njit/jerse/ashe/ASHE.java

```diff
@@ -1,8 +1,8 @@
-package edu.njit.jerse;
+package edu.njit.jerse.ashe;
```

Comment:
I will revert the package change on this PR and create a new PR with the package change.

Reply:
Please don't revert in this PR, as it might mark comments as out-of-date.  Just make the change in a different PR, merge that PR, then pull master into this branch.  Thanks!

Reply:
[PR #18](https://github.com/jonathan-m-phillips/ASHE_Automated-Software-Hardening-for-Entrypoints/pull/18) has addressed this.